pegasus/migrations/76_migrate_channel_sources_to_s3.rb

```diff
@@ -0,0 +1,60 @@
+require 'cdo/hip_chat'
+require 'files_api'
+
+# Migration for moving JSON.parse(row[:value])['levelSource'] and ['levelHtml'] to S3 (/v3/sources API).
+Sequel.migration do
+  up do
+    HipChat.log 'Moving <b>channel</b> sources to S3...'
+    FILE_NAME = 'main.json'
+
+    # For each channel with a level(Source|Html) and no migratedToS3 flag, create a new S3 record and set migratedToS3.
+    batch_update do |row|
```

Comment:
this looks like it batches the reading of the rows but not the writing of the rows. Is that intended?